en/developer_guide/robots.rst

```diff
@@ -0,0 +1,73 @@
+Using robots.txt
+================
+
+As Open Orchestra is a multisite platform, you will have to manage several robots.txt files for a
+single installation. To simplify the maintenance of their content, they can be directly edited in
+the Back Office. But contribution is only the first step. To be available, these information
+require to be dumped in files and to be correctly served to the client.
+
+Creation of ``robots.txt`` files with Open Orchestra is achieved in 3 steps:
+
+- `files contribution`_
+- `files generation`_
+- `files routing`_
+
+.. _files contribution:
+
+Files contribution
+------------------
+Contributing the content of a ``robots.txt`` file is straight forward. As it is specific to each
+website, you can do it in the website edit form, located in the Administration submenu. Fill the 
+matching textarea: the content of the ``robots.txt`` file will be exactly what is written here.
+
+.. _files generation:
+
+Files generation
+----------------
+Once contributed, the content of the ``robots.txt`` is only known from the database. To serve it to
+a client requesting it, two approaches are possible: use a controller action looking in the database
+for the information matching the current site, or serve a previously generated file containing these
+information. To enhance perfomances, Open Orchestra choose the second option. This way, there is no
+intervention of Symfony, PHP and no access to the Datatbase. These static files can also be easily
+cached by a reverse proxy.
+
+So when ``robots.txt`` files are contributed, you have to dump them into physical files. To help you
+to generate them, Open Orchestra provides a console command that must be used on your Front
+installation:
+
+    app/console orchestra:robots:generate [--siteId=SITEID]
+
+With that command, the ``robots.txt`` files of all sites are dumped, or only one if you provide a
+siteId. The generated files are stored in the web directory of your application, to allow them to
+be accessed by clients. Each file is stored in a sub-directory named by the site Id. For instance
+if you have three sites whose id is 'site-1', 'site-2' and 'site-3', the ``robots.txt`` files will
+be dumped here:
+
+- web/site-1/robots.txt
+- web/site-2/robots.txt
+- web/site-3/robots.txt
+
+A good practice is to use that command in a cron, to refresh periodically the content of the files.
+
+.. _files routing:
+
+Files routing
+-------------
+When a client requests a ``robots.txt``, it requests it at the root of the domain. But the files are
+generated somewhere else, in a subfolder named by the site id. So the webserver must be configured to
```

Comment:
As the files are generated in a subfolder, the webserver must be configured to redirect the request to the matching file